runtime/oti/vm_api.h

```diff
@@ -4734,7 +4734,8 @@ preparePinnedVirtualThreadForMount(J9VMThread *currentThread, j9object_t continu
  * @param syncObj object to block/wait on
  * @param isObjectWait if the call is from Object.wait()
  *
- * @return J9_OBJECT_MONITOR_YIELD_VIRTUAL if the can be successfully yielded;
+ * @return syncObj if isObjectWait is false and monitor can acquired;
+ * J9_OBJECT_MONITOR_YIELD_VIRTUAL if the can be successfully yielded;
```

Comment:
monitor can acquired -> monitor can be acquired
Perhaps "monitor" is missing in "if the can be"?